src/vivarium_nih_us_cvd/components/paf_observer.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, List
+
+import pandas as pd
+from vivarium.framework.engine import Builder
+
+
+class PAFObserver:
+
+    configuration_defaults = {
+        "stratification": {
+            "paf": {
+                "exclude": [],
+                "include": [],
+            }
+        }
+    }
+
+    def __init__(self, risk: str, target: str):
+        self.risk = risk
+        self.target = target
+        self.configuration_defaults = self.get_configuration_defaults()
+
+    def __repr__(self):
+        return f"PAFObserver({self.risk}, {self.target})"
+
+    @property
+    def name(self):
+        return f"paf_observer.{self.risk}.{self.target}"
+
+    # noinspection PyAttributeOutsideInit
+    def setup(self, builder: Builder) -> None:
+        self.step_size = builder.time.step_size()
+        self.config = builder.configuration.stratification["paf"]
```

Comment:
Since you only use this in the `setup` method, there's no need to save this to the `self` object.